e2e_tests/docker/indexer-filtered/e2e-write/Dockerfile

```diff
@@ -0,0 +1,42 @@
+ARG GO_IMAGE=golang:1.14.7
+FROM $GO_IMAGE
+ARG CHANNEL=stable
+ARG CI_E2E_FILENAME
+
+RUN echo "Go image: $GO_IMAGE"
+
+# Misc dependencies
+ENV HOME /opt
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y apt-utils curl git git-core bsdmainutils python3 python3-pip make bash libtool libboost-math-dev libffi-dev wget gettext-base
+
+# Setup files for test
+RUN mkdir -p /opt/go/indexer
+RUN mkdir -p /opt/conduit-dir
+RUN mkdir -p /opt/algod-dir
+COPY . /opt/go/indexer
```

Comment:
we'll be removing conduit from this repo? should we download the conduit repo instead?

Reply:
Yes, but when I wrote this the conduit repo didn't exist/didn't have a buildable binary.

We can either wait until everything is ready (which maybe it is now?) and update this or just do this in a followup commit.

I think I prefer following up on this in the future. For example, this PR https://github.com/algorand/indexer/pull/1505 would start failing and would force me to make this change.